backend/workflow_manager/endpoint/db_connector_helper.py

```diff
@@ -0,0 +1,77 @@
+from typing import Any
+
+from google.cloud import bigquery
+from workflow_manager.endpoint.constants import DBConnectionClass, TableColumns
+
+from unstract.connectors.databases.unstract_db import UnstractDB
+
+
+class DBConnectorQueryHelper:
```

Comment:
NIT: It'll be good if these can be moved into the DBConnector base class itself since we see elements of `if cls == <CLASSNAME>`

Reply:
@chandrasekharan-zipstack let this be. Let the create/insert queries be on `database_utils` since its a **service class**. Connector will have only have the `execute_query` method

Reply:
I also believe that moving these queries into the DBConnector classes or its base class could be beneficial.@kirtimanmishrazipstack . It's not necessory but will be good If we have time.

Reply:
This refactor will be done with the multi column insertion changes later
CC: @kirtimanmishrazipstack